test/controllers/income/bbsi/BbsiControllerSpec.scala

```diff
@@ -156,6 +183,10 @@ class BbsiControllerSpec extends PlaySpec
 
         when(sut.bbsiService.bankAccount(any(), any())(any())).thenReturn(Future.successful(Some(bankAccount)))
 
+        val cache = Map(UpdateBankAccountUserChoiceKey -> UpdateInterest)
+
+        when(sut.journeyCacheService.cache(any(), any())(any())).thenReturn(Future.successful(cache))
```

Comment:
For these tests we could get away with returning an empty Map from the cache method (the implementation ignores the response anyway)